{"uuid":"372c5060-e74c-4868-9f13-dc2e7e48bff6","historyId":"44e1325b76ef7a86fab93ffe11c249c","fullName":"tests.AdminTest.testAdminFeatures","labels":[{"name":"package","value":"tests.AdminTest"},{"name":"testClass","value":"tests.AdminTest"},{"name":"testMethod","value":"testAdminFeatures"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"tests.AdminTest"},{"name":"host","value":"Bahaa"},{"name":"thread","value":"20324@Bahaa.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Test Admin Features: Login, Navigate to Admin Page, and Add a New User","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.io.IOException: File not found: user_data.txt","trace":"java.lang.RuntimeException: java.io.IOException: File not found: user_data.txt\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:50)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:146)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:798)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:740)\r\n\tat org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)\r\n\tat org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\nCaused by: java.io.IOException: File not found: user_data.txt\r\n\tat tests.TestDataReader.userDataProvider(TestDataReader.java:20)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:77)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:46)\r\n\t... 25 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1740545327569,"stop":1740545327572}